### ISSUE TITLE
deprecation warnings

### ISSUE THREAD

**Issue by jlozadad** [ANXS/postgresql#322]
This ticket will be used to keep track of deprecation warnings.
```
____________________________________________________
/ TASK [postgresql : PostgreSQL | Install Python pip \
\ installer | yum]                                   /
 ----------------------------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||

[DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|version_compare` instead use `result is version_compare`. This feature 
will be removed in version 2.9. Deprecation warnings can be disabled by setting
 deprecation_warnings=False in ansible.cfg.

```

**Comment by jlozadad** [comment]
`with_items`  changed to `loop`

**Comment by stejoo** [comment]
Above commit & pull-request fixes these deprecation warnings (`version_compare` as filter instead of test).

When I have spare time I might have a go at the loops, but at the moment they work without issue so I'll probably have to soldier on with my other duties for now.

**Comment by gclough** [comment]
Fixed by #387
